src/core/api/index.js

```diff
@@ -59,12 +60,20 @@ export function callApi({
     .then((response) => (schema ? normalize(response, schema) : response));
 }
 
-export function search({ api, page, query, auth = false }) {
+export function categories({ api }) {
+  return callApi({
+    endpoint: 'addons/categories',
+    schema: { results: arrayOf(categorySchema) },
+    state: api,
+  });
+}
+
+export function search({ api, page, query, auth = false, category, addonType }) {
```

Comment:
I think I'd like this to be more generic for the extra filters. Essentially thinking of it as `search({ query, page, filters })` and `filters` would have the category/addon type in it. Once that's done it might be nicer to wrap this in a `category()` function so it's clear how to get category listings.

```js
export function category({ category, addonType, filters = {}, ...args }) {
  return search({ ...args, filters: { ...filters, category, addonType } });
}